packages/formation/sass/base/_b-variables.scss

```diff
@@ -12,9 +12,16 @@
 // ~uswds/src/stylesheets/core/defaults" or
```

Comment:
Just a note: any of the lines indicated to be removed were moved in more logical groupings.